examples/Azure_IoT_Central_Arduino_Nano_RP2040_Connect/Azure_IoT_PnP_Template.h

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/*
+ * Azure_IoT_PnP_Template.cpp implements the IoT Plug and Play template
+ * specific for the Arduino Nano RP2040 Connect board.
+ */
+ 
+#ifndef AZURE_IOT_PNP_TEMPLATE_H
+#define AZURE_IOT_PNP_TEMPLATE_H
+
+#include "AzureIoT.h"
+
+/*
```

Comment:
nit: not sure if this was done on purpose or not, but just for future reference (since this won't actually be generated doxygen docs). Doxygen comments start with two stars to separate them from general, multiline comments:
```suggestion
/**
```

Reply:
don't actually have to make this change.

Reply:
Good to know, thanks for pointing it out!